packages/app/src/app/pages/Sandbox/Editor/Header/PrivacyTooltip/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useOvermind } from 'app/overmind';
+import { ThemeProvider } from 'styled-components';
+import Tooltip from '@codesandbox/common/lib/components/Tooltip';
+import theme from '@codesandbox/common/lib/design-language/theme';
+import { Container, Text, Link, Select } from './elements';
+import * as Icons from './icons';
+
+export const PrivacyTooltip = () => {
+  const {
+    actions: {
+      workspace: { sandboxPrivacyChanged },
+    },
+    state: {
+      user,
+      editor: {
+        currentSandbox: { privacy },
+      },
+    },
+  } = useOvermind();
+
+  const config = {
+    0: {
+      description: 'Everyone can see this Sandbox',
+      icon: <Icons.Public />,
+    },
+    1: {
+      description:
+        'Only people with a private link are able to see this Sandbox',
+      icon: <Icons.Unlisted />,
+    },
+    2: {
+      description: 'Only you can see this Sandbox.',
+      icon: <Icons.Private />,
+    },
+  };
+
+  const onChange = event => {
+    const value = event.target.value;
+    sandboxPrivacyChanged(Number(value) as 0 | 1 | 2);
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Container>
+        <Tooltip
+          delay={0}
+          interactive
+          content={
+            <>
+              <Text size="3" marginBottom={4}>
+                {!user || !user.subscription ? (
```

Comment:
```suggestion
                {user?.subscription ? (
```

This might fix it?

Reply:
Took this in

Reply:
Whoop back to: `Cannot read property 'subscription' of null`